web-ng/src/app/app.component.ts

```diff
@@ -25,4 +27,15 @@ import { Component } from '@angular/core';
   templateUrl: './app.component.html',
   styleUrls: ['./app.css'],
 })
-export class AppComponent {}
+export class AppComponent {
+  public constructor(@Inject(DOCUMENT) private doc: Document) {
+    this.initGoogleMap();
+  }
+  private initGoogleMap(): void {
```

Comment:
Is hack the DOM necessary? Is there a more robust way to import the script with a dynamic API key?